# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: security/sandbox/chromium/sandbox/linux/system_headers/arm_linux_ucontext.h
# Commit: 370c40e49230
# Full Hash: 370c40e4923073a87845b57c1f4851bacd9663da
# Author: Bob Owen <bobowencode@gmail.com>
# Date: 2019-06-12 21:42:35
# Regressor Bug: 1552160
# File Overlap Count: 2
# Description:
#   Bug 1552160 Part 1: Roll-up of chromium sandbox update and mozilla patches to get a running browser. r=jld,aklotz,tjr,bobowen
#   
#   This updates security/sandbox/chromium/ files to chromium commit 84108231f6e6e0772fb9a4643679ce76aa771e67.
#   
#   Existing and new patches applied from security/sandbox/chromium-shim/patches/with_update/ to give a compiling and mostly working browser.
# ==============================================================================

diff -r bd5ecd72f76c -r 370c40e49230 security/sandbox/chromium/sandbox/linux/system_headers/arm_linux_ucontext.h
--- a/security/sandbox/chromium/sandbox/linux/system_headers/arm_linux_ucontext.h	Wed Jun 12 21:55:59 2019 +1200
+++ b/security/sandbox/chromium/sandbox/linux/system_headers/arm_linux_ucontext.h	Wed Jun 12 11:10:48 2019 +0100
@@ -7,10 +7,6 @@
 
 #include <stddef.h>
 
-#if !defined(__BIONIC_HAVE_UCONTEXT_T)
-#if !defined(__native_client_nonsfi__)
-#include <asm/sigcontext.h>
-#else
 // In PNaCl toolchain, sigcontext and stack_t is not defined. So here declare
 // them.
 struct sigcontext {
@@ -43,7 +39,6 @@
   size_t ss_size;
 } stack_t;
 
-#endif
 
 // We also need greg_t for the sandbox, include it in this header as well.
 typedef unsigned long greg_t;
@@ -62,8 +57,4 @@
   unsigned long uc_regspace[128] __attribute__((__aligned__(8)));
 } ucontext_t;
 
-#else
-#include <sys/ucontext.h>
-#endif  // __BIONIC_HAVE_UCONTEXT_T
-
 #endif  // SANDBOX_LINUX_SYSTEM_HEADERS_ARM_LINUX_UCONTEXT_H_